packages/node/src/handlers.ts

```diff
@@ -121,13 +125,17 @@ async function updateHandler(node: DRPNode, data: Uint8Array, sender: string) {
   data: { id: string, operations: {nonce: string, fn: string, args: string[] }[] }
   operations array contain the full remote operations array
 */
-async function syncHandler(node: DRPNode, sender: string, data: Uint8Array) {
+export async function syncHandler(
+	node: DRPNode,
+	sender: string,
+	data: Uint8Array,
+) {
```

Comment:
should add Promis<boolean>
```suggestion
): Promise<boolean> {
```

Reply:
added